Documents/OrderingRules.md

```diff
@@ -0,0 +1,64 @@
+# Ordering Rules
+
+## The Abstract
+
+Rules which enforce a standard ordering scheme for code contents.
+
+## Rules
+
+According to the [StyleCop Rules Documentation](http://stylecop.soyuz5.com/Ordering%20Rules.html) the ordering is as follows.
+
+Within a class, struct or interface: (SA1201 and SA1203)
+
+* Constant Fields
+* Fields
+* Constructors
+* Finalizers (Destructors)
+* Delegates
+* Events
```

Comment:
**Third reviewer.** Events are declared LIKE fields, but they AREN'T fields. The thing here is try to follow Microsoft rules. And in Microsoft code, Events goes between Properties and Methods.

In other order, ALL the team accept this except you. So I'm not only the Third Reviewer, I'm the voice of the team: "This PR must be merged, accepted and used."